junebug/command_line.py

```diff
@@ -61,6 +61,14 @@ def parse_arguments(args):
     parser.add_argument(
         '--amqp-password', '-amqppass', dest='amqp_pass', type=str,
         help='The password to use for the amqp auth. Defaults to "guest"')
+    parser.add_argument(
+        '--inbound-message-ttl', '-ittl', dest='inbound_message_ttl', type=int,
+        help='The maximum time allowed to reply to a message. Defaults to '
+        '"60" seconds')
+    parser.add_argument(
+        '--inbound-message-prefix', '-imp', dest='inbound_message_prefix',
+        type=str, help='The default redis key prefix to use for storing '
+        'inbound messages. Defaults to "inbound_messages"')
```

Comment:
Do we really want the inbound message prefix to be configurable? If the top-level prefix is already configurable via the redis config, it seems better for this part to just be constant?